docs/zq/README.md

```diff
@@ -1,15 +1,721 @@
-## TBD
-
-This doc will give the zq overview and point to the tooling, language,
-and reference docs.  **issue 3604**
+# zq - command-line Zed query
 
 ## TL;DR
 
-Zed is a pipeline-style search and analytics language for querying
-super-structured data in files, over HTTP, in S3 storage, or in a
-[Zed data lake](../zed/README.md).
+> `zq` is a command-line tool that uses the [Zed language](language.md)
+for pipeline-style search and analytics language.  `zq` can query a variety
+of data formats in files, over HTTP, or in S3 storage
+It is particularly fast when you operating over the Zed-native ZNG format.
+>
+> The `zq` design philosophy blends the query/search-tool approach
+of `jq`, `awk`, `grep` with the command-line, embedded database approach
+of `sqlite` and `duckdb`.
+>
+> Most examples of Zed are provided as `zq` one liners, but this is just
+one way of using Zed as anything running at desktop-scale as a `zq` command
+can be run at cloud scale using a Zed lake.
```

Comment:
This doesn't feel like TL;DR material to me.

Reply:
I'll just take it out